### DISCUSSION TITLE
Pulling RTMA sub-hourly (rapid update)

### DISCUSSION THREAD

**Discussion by prosaicpudding** [blaylockbk/Herbie#202] [General]
I am working with the new RTMA model template, and was wondering if there is a way to get the sub hourly products. I didn't see it in the documentation, but I didn't see anything there. Is this possible yet?

**Comment by prosaicpudding** [comment]
Ah,  from source it looks like this is currently in development, but has not been released yet. Is that correct?

**Reply by blaylockbk** [reply]
Correct. I haven't had too much time to fully implement RTMA; I am planning on releasing what I have so far in the next week or two...just need to find a free afternoon to do the release.

**Reply by prosaicpudding** [reply]
Thanks. In the meantime, I did figure out how to pull mid-release versions, which resolves most of the issue for me. However, I did notice a couple things that I figure might save you a little time when you get around to it. 

First, the path in the template looks like it's not quite correct (or at least it was throwing an error for me). The below version works, except that it will by default pull from aws which, apparently, doesn't include the index files. I'm sure you have a way to force it to check the next source when those are missing, but it wasn't immediately clear to me. 

Anyway. Hope this helps. If you'd prefer a PR, let me know and I can do that. It just seemed like overkill for such a tiny suggestion. 

Cheers, and thanks for all your work on this project!

``` rtma.py
class rtma_ru:
    def template(self):
        self.DESCRIPTION = "CONUS Rapid Update Real-Time Mesoscale Analysis (RTMA)"
        self.DETAILS = {
            "NOMADS product description": "https://www.nco.ncep.noaa.gov/pmb/products/rtma/",
            "Amazon Open Data": "https://registry.opendata.aws/noaa-rtma/",
        }
        self.PRODUCTS = {
            "anl": "Analysis",
            "err": "Analysis Forecast Error",
            "ges": "Forecasts",
        }


        PATH = f"rtma2p5_ru.{self.date:%Y%m%d}/rtma2p5_ru.t{self.date:%H%M}z.2dvar{self.product}_ndfd.grb2"
        self.SOURCES = {
            "aws": f"https://noaa-rtma-pds.s3.amazonaws.com/{PATH}",
            "nomads": f"https://nomads.ncep.noaa.gov/pub/data/nccf/com/rtma/prod/{PATH}",
        }
        
        self.LOCALFILE = f"{self.get_remoteFileName}"
```
